cmake/scripts/linux/platform/X11.cmake

```diff
@@ -0,0 +1,2 @@
+set(PLATFORM_REQUIRED_DEPS OpenGl EGL X XRandR LibDRM)
```

Comment:
most people install libgl-mesa-dev which includes libgl1-mesa-glx. that may be the reason the issue has not come up yet.
not sure if a proper solution is worth the effort. maybe just put a comment in the code that states that glx is a package by itself